scripts/install_coturn_on_aws_ec2.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+#####################################################################################
+############## purpose                 : install coturn as a server          ########
+############## verification on aws     : tested & verified on aws ec2        ########
+############## platform                : aws ec2                             ########
+############## aws ubuntu version      : ubuntu 22.04 LTS                    ########
+############## coturn base version     : turnserver-4.5.2.tar.gz             ########
+############## dependent tools         : installs prometheus client as well  ########
+############## file permissions        : chmod 777 install_coturn_on_aws_ec2.sh #####
+############## run command             : ./install_coturn_on_aws_ec2.sh      ########
+############## developer               : hariprasad.t@samsung.com            ########
+#####################################################################################
+
+echo "--------> this script installs coturn server version coturn-4.5.2 on aws ec2 instance..."
+
+echo "--------> create user turnserver..."
+sudo adduser --gecos "" --disabled-password turnserver
+
+echo "--------> updating packages..."
+sudo DEBIAN_FRONTEND=noninteractive apt-get -y update
+sudo DEBIAN_FRONTEND=noninteractive apt-get -y upgrade
+
+### install all dependent packages
+echo "--------> installing dependent packages..."
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y gcc make openssl-dev build-essential pkg-config libsystemd-dev musl-dev sqlite
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y libssl-dev libsqlite3-dev libevent-dev libpq-dev libmysqlclient-dev libhiredis-dev libmicrohttpd-dev
+
+### download prometheus client libraries
+echo "--------> installing prometheus client..."
+wget https://github.com/digitalocean/prometheus-client-c/releases/download/v0.1.3/libprom-dev-0.1.3-Linux.deb
+wget https://github.com/digitalocean/prometheus-client-c/releases/download/v0.1.3/libpromhttp-dev-0.1.3-Linux.deb
+sudo dpkg -i prometheus-client/libprom-dev-0.1.3-Linux.deb
+sudo dpkg -i prometheus-client/libpromhttp-dev-0.1.3-Linux.deb
+
+### download coturn source code
+echo "--------> downloading turnserver v4.5.2 ..."
+wget https://coturn.net/turnserver/v4.5.2/turnserver-4.5.2.tar.gz
```

Comment:
I do not know who supports this site - this is not an official source IMHO
Pls change it to `https://github.com/coturn/coturn/archive/refs/tags/4.6.2.tar.gz` which is a tagged release on github

Also, this will make it much easier to change the version

Actually, I would make a version to be an optional argument to the script while 4.6.2 is default (current last tag)

Reply:
Sure, I will do update the PR based on your review comments. Thanks.

Reply:
1. Script is now parameterized. We can pass pass coturn version as an argument. If not provided, then it considers 4.6.2 as default version.
2. Verified this script on AWS EC2 instance also.